app/models/area.rb

```diff
@@ -1,4 +1,44 @@
 class Area < ActiveRecord::Base
   include GeoModule
   after_create :generate_geohash
+
+  def places_belong_to_this_area_filterd
+    neighbors = GeoHash.neighbors(self.geohash).unshift(self.geohash)
+    return Place.where(geohash: detect_neighbors_indexes(neighbors))
+  end
+
+  # 強制的に周辺エリア全域のお店を抽出したい場合にはこちらを利用
```

Comment:
__[rubocop]__
* Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)